indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -506,6 +523,8 @@ protected void reduce(
 
       final Interval interval = config.getGranularitySpec().bucketInterval(bucket.time).get();
 
+      ListeningExecutorService persistExecutor = null;
```

Comment:
Most places where new executor services are created are using listener services right now, mostly because they provide more functionality and realizing later that you should have made it a listening service is a pain. Plus handy things like Futures.allAsList only work on listenable futures. So in essence the practice of always using a listening service makes the code easier to add stuff to later at the cost of depending on an external library instead of just java.* classes. Is there any reason you know of why using listening executor services by default is a bad idea?